rllib/core/rl_trainer/trainer_runner.py

```diff
@@ -71,10 +64,12 @@ def __init__(
         self._is_local = scaling_config.num_workers == 0
         self._trainer = None
         self._workers = None
+        self._is_shut_down = False
```

Comment:
why do we have this flag?

Reply:
To prevent calling backend executor.shutdown 2x

Reply:
I need this so that if a user calls shutdown then when del is called the backend executor isn't shutdown twice causing an error

Reply:
doesn't backend have something like `is_alive` or some flag which tells you whether it's shutdown or not? 
If not please write a comment here to say why you did this.

Reply:
done